ocs_ci/templates/workloads/fio/fill_pool_job.yaml

```diff
@@ -0,0 +1,40 @@
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: fill-pool-job
+spec:
+  backoffLimit: 0
+  template:
+    metadata:
+      labels:
+        app: fill-pool
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: filler
+        image: quay.io/ocsci/busybox:latest
+        imagePullPolicy: IfNotPresent
+        command:
+          - sh
+          - -c
+          - |
+            echo "Filling PVC with zeros..."
+            dd if=/dev/zero of=/mnt/fill/testfile bs=${BLOCK_SIZE:-1M}
```

Comment:
isn't input file (if) supposed to be user defined? here it's set as zero

Reply:
Fixed